tests/unit/providers/test_otari_provider.py

```
@@ -44,6 +48,13 @@ def _mock_otari_client() -> MagicMock:
     return client
 
 
+def _setup_messages_mock(provider: OtariProvider, sample_response: dict[str, Any]) -> AsyncMock:
```

Ruff (0.15.15)

[error] Undefined name `Any`

(F821)